files/default/neutron-ha-tool.py

```diff
@@ -579,6 +579,16 @@ def list_agents(qclient, agent_type=None):
     return resp['agents']
 
 
+def list_alive_agents(qclient, agent_type=None):
+    resp = qclient.list_agents()
+    LOG.debug("list_agents: %s", resp)
+    if agent_type:
+        return [agent for agent in resp['agents']
+                if agent['agent_type'] == agent_type and
+                agent['alive'] is True]
```

Comment:
This feels wrong: we should check for `alive` regardless of whether `agent_type` is `None` or not.